src/mlpack/tests/load_save_test.cpp

```diff
@@ -1454,8 +1456,10 @@ TEST_CASE("NontransposedCSVDatasetInfoLoad", "[LoadSaveTest]")
   {
     arma::mat one, two;
     DatasetInfo info;
-    data::Load(testFiles[i], one, true, false); // No transpose.
-    data::Load(testFiles[i], two, info, true, false);
+  if (!data::Load(testFiles[i], one, true, false)) 
+    FAIL("Cannot load dataset"); 
+  if (!data::Load(testFiles[i], two, info, true, false)) 
+    FAIL("Cannot load dataset"); 
```

Comment:
Would it be possible to add the dataset name here?
testFiles is a vector of strings so the ```FAIL()``` message could include the dataset name.
```testFiles[i]``` should give the name of the dataset that failed.

Reply:
We can put the value of testFiles[i] in a string and then pass the string to the fail message, does that sound good?

Reply:
Yup. That sounds good. 

> And sorry to trouble you with all the style checks.

Not an issue. 👍🏼

Reply:
Hi. I think I was not clear enough when I pointed out the indentation issues.
I can only mark a line so I pointed the start of the line.
But the entire block of code is actually indented wrong. 
You can get a better idea when you see the code diff [here](https://github.com/mlpack/mlpack/pull/2717/files).

Reply:
Implemented that failed message as discussed...

Reply:
Working on it...

> Hi. I think I was not clear enough when I pointed out the indentation issues.
> I can only mark a line so I pointed the start of the line.
> But the entire block of code is actually indented wrong.
> You can get a better idea when you see the code diff [here](https://github.com/mlpack/mlpack/pull/2717/files).

Reply:
I think it's done now.